### PR TITLE
Update readme with explanation of some dependencies

### PR DESCRIPTION
**Pull request by gbzaid2**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,11 @@
 # FEC Greenfield
 
 ...
+A list of Dependencies and their intended use.
+
+React-router-dom: Handles the routing, alternative to express
```
Comment (jrd404) [inline README.md:6]: I slacked out a guide to writing Markdown (the formatting syntax used by files ending in `.md`).

Can you read up on that and take another stab at this.

Just have to add some simple styling things, perhaps listing dependencies as an ordered list and formatting each dependency in backticks so they appear in "code" formatting.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,11 @@
...
+Bulma: CSS framwork
```
Comment (jrd404) [inline README.md:7]: i.e. 

## A List of Dependencies and their Intended Use

1. `bulma`: A pure CSS framework
2. `node-sass`: A SASS library for pre-compiling CSS

File: README.md [hunk 3]
```diff
@@ -1,3 +1,11 @@
 # FEC Greenfield
 
 ...
+A list of Dependencies and their intended use.
+
+React-router-dom: Handles the routing, alternative to express
+Bulma: CSS framwork 
+eslint: cleans up code with "bad style" - you can add your own style guidelines to eslint and have eslint enforce it before you push your code
```
Comment (jrd404) [inline README.md:8]: Maybe getting a bit more specific about what `eslint` is than "bad style"?

Learning Markdown is super helpful and makes for the READMEs that we all appreciate in other open-source repos :)

**Comment by tonyguo665** [comment]
image

**Comment by jrd404** [comment]
#16

**Pull request merged**